### ISSUE TITLE
Warning message comes up even though I am using Pyinstaller 4.10!

### ISSUE THREAD

**Issue by mcagriaksoy** [brentvollebregt/auto-py-to-exe#259]
Hi, I got the message :
![image](https://user-images.githubusercontent.com/20202577/161539590-0c8d5d88-1d4d-4a21-95e2-2504f0db4539.png)

and it causes from the file: utils.py line: 72 somehow it goes in the condition line 73 should not!
![image](https://user-images.githubusercontent.com/20202577/161539542-302bcb9b-bb69-42da-a494-1a73b4e34ea7.png)

**Comment by brentvollebregt** [comment]
Can you execute this and let me know what the output is:

```python
from PyInstaller import __version__ as pyinstaller_version
print(f'pyinstaller_version: {pyinstaller_version}')
print(f'float(pyinstaller_version): {float(pyinstaller_version)}')
```

**Comment by natura4917** [comment]
pyinstaller_version: 4.10
float(pyinstaller_version): 4.1

**Comment by brentvollebregt** [comment]
@mcagriaksoy or @natura4917, can one of you (or someone else looking at this) checkout the branch `feature/259-fix-pyinstaller-4.10-warnings` and check this is fixed please?

PR is at https://github.com/brentvollebregt/auto-py-to-exe/pull/260

**Comment by mcagriaksoy** [comment]
congratz. it is fixed. thanks for quick response.
![image](https://user-images.githubusercontent.com/20202577/161824653-5009d080-cf54-4365-a54a-3eeb1b56a4c2.png)
